examples/tf-azurerm-vnet/variables.tf

```diff
@@ -0,0 +1,28 @@
+variable "prefix" {
+  description = "Default prefix to use with your resource names."
+  default = "myapp"
+}
+
+variable "location" {
+  description = "The location/region where the core network will be created. The full list of Azure regions can be found at https://azure.microsoft.com/regions"
+}
+
+variable "address_space" {
+  description = "The address space that is used by the virtual network."
+  default     = "10.0.0.0/16"
+}
+
+variable "dns_servers" {
+  description = "The DNS servers to be used with vNet"
```

Comment:
might be worth mentioning this defaults to Azure DNS (from memory, I may be wrong) if these aren't specified?